src/features/game/components/Revealed.tsx

```diff
@@ -24,41 +24,74 @@ export const Revealed: React.FC<{ onAcknowledged?: () => void }> = ({
 
   const items = getKeys(gameState.context.revealed?.inventory ?? {});
   const sfl = Number(gameState.context.revealed?.balance ?? 0);
+  const seasonalTicket =
+    gameState.context.revealed?.inventory[getSeasonalTicket()];
 
+  const ticketImage = () => {
+    return (
+      <img
+        src={ITEM_DETAILS[getSeasonalTicket()].image}
+        className="mb-2"
+        onLoad={(e) => setImageWidth(e.currentTarget)}
+        style={{
+          opacity: 0,
+        }}
+      />
+    );
+  };
+
+  const ticketText = () => {
+    return (
+      <p className="text-center text-sm mb-2">{`and 1x ${getSeasonalTicket()}`}</p>
+    );
+  };
   return (
     <>
       <div className="flex flex-col items-center p-2">
         <p className="text-center text-base mb-2">Congratulations!</p>
+
         {sfl > 0 && (
           <>
-            <img
-              src={token}
-              className="mb-2"
-              style={{
-                width: `${PIXEL_SCALE * 10}px`,
-              }}
-            />
-            <p className="text-center text-sm mb-2">{`You found ${sfl} SFL`}</p>
-          </>
-        )}
-
-        {items.length > 0 &&
-          items.map((name, index) => (
-            <div
-              key={`${name}-${index}`}
-              className="flex flex-col items-center"
-            >
+            <div className="flex flex-wrap justify-center items-center mb-2 space-x-2 mt-1">
               <img
-                src={ITEM_DETAILS[name as InventoryItemName].image}
+                src={token}
                 className="mb-2"
                 onLoad={(e) => setImageWidth(e.currentTarget)}
-                style={{
-                  opacity: 0,
-                }}
               />
-              <p className="text-center text-sm mb-2">{`You found ${gameState.context.revealed?.inventory[name]} x ${name}`}</p>
+              {seasonalTicket && ticketImage()}
             </div>
-          ))}
+
+            <p className="text-center text-sm mb-2">{`You found ${sfl} SFL`}</p>
+
+            {seasonalTicket && ticketText()}
+          </>
+        )}
+
+        {items.length > 0 &&
+          items.map(
+            (name, index) =>
+              name != getSeasonalTicket() && (
```

Comment:
Do we need to support the specific "Seasonal Ticket"?

I would suggest changing this logic to support all manner of inventory items, instead of adding specific case logic into the component.

Example:

```
{/* Images */}
        <div className="flex flex-wrap justify-center items-center mb-2 space-x-2 mt-1">
          {sfl > 0 && (
            <img
              src={token}
              className="mb-2"
              onLoad={(e) => setImageWidth(e.currentTarget)}
            />
          )}

          {items.map((name, index) => (
            <img
              src={ITEM_DETAILS[name as InventoryItemName].image}
              className="mb-2"
              onLoad={(e) => setImageWidth(e.currentTarget)}
              style={{
                opacity: 0,
              }}
            />
          ))}
        </div>

        {/* Text*/}
        <div>
          {sfl > 0 && (
            <p className="text-center text-sm mb-2">{`You found ${sfl} SFL`}</p>
          )}

          {items.map((name, index) => (
            <p className="text-center text-sm mb-2">{`You found ${gameState.context.revealed?.inventory[name]} x ${name}`}</p>
          ))}
        </div>
```

E.g.